src/platform/packages/shared/kbn-css-utils/public/focus_ring_css.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+import type { UseEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+
+export const removeEuiFocusRing = css`
+  outline: none;
+  &:focus-visible {
+    outline-style: none;
+  }
+`;
+
+export const passDownFocusRing = (euiTheme: UseEuiTheme['euiTheme'], target: string) => css`
```

Comment:
This is only imported from a single place, `./src/platform/packages/shared/kbn-unified-field-list/src/components/field_item_button/field_item_button.tsx`. Can we just inline the styles there, where it's needed instead of having an additional module?